src/test/java/core/basesyntax/strategy/AdditionStrategyImplTest.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax.strategy;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Operation;
+import core.basesyntax.model.TransactionDto;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class AdditionStrategyImplTest {
+    private static OperationStrategy additionStrategy;
+    private static final TransactionDto testDto1 = new TransactionDto(Operation.BALANCE,
+            new Fruit("banana"), 20);
+    private static final TransactionDto testDto2 = new TransactionDto(Operation.SUPPLY,
+            new Fruit("apple"), 100);
+    private static final TransactionDto testDto3 = new TransactionDto(Operation.RETURN,
```

Comment:
```suggestion
    private static final TransactionDto TEST_DTO_1 = new TransactionDto(Operation.BALANCE,
            new Fruit("banana"), 20);
    private static final TransactionDto TEST_DTO_2 = new TransactionDto(Operation.SUPPLY,
            new Fruit("apple"), 100);
    private static final TransactionDto TEST_DTO_3 = new TransactionDto(Operation.RETURN,
```
same in FruitServiceImplTest and ReductionStrategyTest

Reply:
done